### PR TITLE
UNO 235 add ckeditor and simulator libs

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/UNO-235

`CKeditor` and `simulator` were moved from `tao-core`, `tao-core-ui-fe`,` tao-item-runner-qti-fe` to `tao-core-shared-libs-fe` to reduce duplication of code
How to update CKeditor in `lib/ckeditor/README.md`

Related PRs:
https://github.com/oat-sa/ckeditor-dev/pull/15
https://github.com/oat-sa/tao-core-ui-fe/pull/123
https://github.com/oat-sa/extension-tao-devtools/pull/178
https://github.com/oat-sa/tao-item-runner-qti-fe/pull/73
Related PRs (**they will be updated**):
https://github.com/oat-sa/tao-core/pull/2498

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@oat-sa/tao-core-shared-libs",
+    "version": "0.1.0",
+    "description": "TAO Frontent Core Shared Libraries",
+    "files": [
+        "dist",
+        "src",
+        "lib"
+    ],
```
Comment (krampstudio) [inline package.json:9]: is it accurate ?

Comment (bziondik) [reply package.json]: fixed

**Comment by bziondik** [comment]
> Looks good overall to me.
> Can you please mention in the README the format of the modules. Because for some packages (tao-core-sdk) the sources are using ES module and we compile them to AMD version.
> Here the sources are in AMD, so it should be mentioned.

@krampstudio  Updated readme

**Comment by krampstudio** [comment]
> > Looks good overall to me.
> > Can you please mention in the README the format of the modules. Because for some packages (tao-core-sdk) the sources are using ES module and we compile them to AMD version.
> > Here the sources are in AMD, so it should be mentioned.
> 
> @krampstudio Updated readme

Then the overall approach looks good to me

**Pull request merged**